content/sensu-core/2.0/reference/assets.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Assets"
+description: "The assets reference guide."
+weight: 1
+version: "2.0"
+product: "Sensu Core"
+platformContent: true
+menu:
+  sensu-core-2.0:
+    parent: reference
+---
+
+# Assets
+
+## What are Assets?
+
+Sensu Assets are resources that checks can specify as dependencies. When an
```

Comment:
s/Assets/**assets**/ 

I'm recommending we stick with one way of highlighting new concepts. Capitalization can get confusing with time, so on https://github.com/sensu/sensu-docs-site/pull/214 I recommend we **bold new terms when introduced**. Then they can be plaintext going forward.